generators/app/prompts.js

```diff
@@ -115,6 +117,26 @@ module.exports = [
     default: false,
     when: props => props.ui !== 'ionic'
   },
+  {
+    type: 'confirm',
+    name: 'i18n',
+    message: 'Do you want language internationalization support?',
+    default: true
+  },
+  {
+    type: 'checkbox',
+    name: 'i18n-langs',
+    message: 'Which languages do you want to support?',
```

Comment:
maybe a validator should be added here to ensure at least 1 language is selected?

Reply:
Does it support validator? I've validated it in index.js

```js
  configuring() {
    if (!this.props['i18nLanguages']) {
      this.props['i18nLanguages'] = ['en-US'];
    }
```

Reply:
Yes, using the `validate` property, see https://github.com/SBoudrias/Inquirer.js/#objects

I missed you fallback though, so either one is good.